src/main/java/at/hannibal2/skyhanni/config/features/garden/laneswitch/LaneSwitchSoundSettings.java

```diff
@@ -23,6 +23,11 @@ public class LaneSwitchSoundSettings {
     @ConfigEditorButton(buttonText = "Test")
     public Runnable testSound = FarmingLaneFeatures::playUserSound;
 
+    @Expose
+    @ConfigOption(name = "Repeat Duration", desc = "Change in how many ticks the sound will repeat. Change to 20 for only once per second.")
```

Comment:
This description sounds slightly weird but rn idk how to word it better

Reply:
Change how often the sound should be repeated in ticks. Change to 20 for only once per second.